indico/modules/categories/client/js/components/CategoryStatistics.jsx

```diff
@@ -38,12 +38,15 @@ CategoryStatistics.propTypes = {
 };
 
 function getGraphData(data) {
-  const serialize = statsData =>
-    Object.entries(statsData)
+  const serialize = statsData => [
+    [moment(data.minYear - 1, 'YYYY'), null],
+    ...Object.entries(statsData)
       .filter(([year]) => year <= data.maxYear + 3 && year >= data.minYear)
-      .map(([year, number]) => [moment(year, 'YYYY'), number]);
+      .map(([year, number]) => [moment(year, 'YYYY'), number]),
+    [moment(data.maxYear + 4, 'YYYY'), null],
```

Comment:
This won't work, because we only take up to 3 years after the current year depending on how much data there is

Reply:
What do you mean? I just went here for a simple solution instead of actually checking what the max date in the returned list is.